api/loader/fileloader_test.go

```diff
@@ -596,14 +535,11 @@ func TestLoaderHTTP(t *testing.T) {
 	}
 
 	l1 := NewFileLoaderAtRoot(MakeFakeFs(testCasesFile))
-	if "/" != l1.Root() {
-		t.Fatalf("incorrect root: '%s'\n", l1.Root())
-	}
+	require.Equal("/", l1.Root())
```

Comment:
Please move all the `require` changes out to its own PR. We can get that in quickly and rebase.

Reply:
Moved `require` changes into #4661